fmriprep/interfaces/reports.py

```diff
@@ -125,3 +161,18 @@ def _generate_segment(self):
         return FUNCTIONAL_TEMPLATE.format(stc=stc, sdc=sdc, registration=reg,
                                           output_spaces=', '.join(self.inputs.output_spaces),
                                           confounds=', '.join(self.inputs.confounds))
+
+
+class AboutSummaryInputSpec(BaseInterfaceInputSpec):
+    version = Str(desc='FMRIPREP version')
+    command = Str(desc='FMRIPREP command')
+    # Date not included - update timestamp only if version or command changes
+
+
+class AboutSummary(SummaryInterface):
```

Comment:
This interface captures the most recent version/command line used. The timestamp is only updated if either changes. Seem reasonable @chrisfilo?